job-descriptions/fullstack-engineer.md

```diff
@@ -0,0 +1,55 @@
+
+## Qualifications
+
+We are looking for a full stack engineer who has strong fundamentals in good software development techniques, design patterns and best practices. In your career you have worked with and refactored existing code bases. You are both productive and pragmatic because you believe software is only useful if it is used. Collaborating with small high performing teams, you have built and deployed production-ready software that delivers value to customers.
+
+* Strong working knowledge with API design and architecture.
+* Experience building and testing end-to-end applications (e.g. unit testing, integration and e2e testing).
+* Good knowledge of Git in particular and other version control systems in general.
+* Experience building single page applications or progressive web apps and Web APIs.
+* Understanding of core web technologies (HTTP, HTML, CSS, JavaScript)
+* Strong experience with at least one web framework such as React, Angular, Vue.js, Polymer or Closure.
+* Strong experience in at least one server side language such as Go, Java, Python, PHP, Ruby.
+* Experience in SQL, Postgres or other transactional, relational databases. Able to model and design schemas.
+* Have worked with containers in production environments.
+
+## Nice-to-haves
+
+* Direct experience with any one of these technologies: ES6+, Go, GraphQL, TypeScript, Docker, Kubernetes.
+* Experience working on small high-performing teams, preferably tech startups.
+* Comfortable using a cloud computing platform such as AWS, GCP or Azure.
+* Published blog posts and/or tech talks about your work.
+* Contributions to open source projects.
+
+## Learn more about us
+
+To create a product that serves the needs of all developers, we are building a diverse [all-remote team](https://about.sourcegraph.com/company/remote) that is [distributed across the world](https://about.sourcegraph.com/company/team). Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+
+We provide [competitive compensation](https://about.sourcegraph.com/handbook/people-ops/compensation) and [practical benefits](https://about.sourcegraph.com/handbook/people-ops/benefits-and-perks) to keep you happy and healthy so that you can do your best work.
+
+Learn more about what it is like to work at Sourcegraph by reading [our handbook](https://about.sourcegraph.com/handbook/).
+
+## Interview process
+
+1. You [apply here](TODO).
```

Comment:
This needs a link

Reply:
Added to TODO here: https://github.com/sourcegraph/interviews/pull/136#issue-468856668